performance/server/src/main/java/org/hawkular/apm/performance/server/Service.java

```diff
@@ -35,7 +34,7 @@
 
     private String name;
 
-    private Map<String, String> calledServices = new HashMap<String, String>();
+    private Map<String, String> calledServices;
```

Comment:
Is this initialized in a different point ?
I can't see it on the patch and I see a reference of this one in some code below.